_lessons/17-adding-placeholder-text.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Adding Placeholder Text
+lesson: 17
+layout: default
+---
+
+Now, we are ready to begin filling in the layout with some placeholder text. First, let's add some headings to the header. In the last section, we learned that the tags `h1` through `h6` are used for headings. We'll use `h1` for the biggest title, and `h2` for a subtitle. 
+
+```
+<div id="header">
+  <h1>My First Webpage</h1>
+  <h2>My Web Development Journey Begins</h2>
+</div>
+```
+
+Next, let's add a few fake links to the sidebar using the `a` tag, which stands for anchor tag. Eventually, we can use these to link to more pages in our website, or other sites we like. For now, they will be placeholders so that we can style them. 
+
+The `href` attribute is used to specify which URL it links to. It's important to set this to something in our placeholders, because links that don't go anywhere will actually look different on the page. We want to see what links that go somewhere look like. 
```

Comment:
````suggestion
Next, let's add a few fake links to the sidebar using the `a` tag, which stands for "anchor" tag. Eventually, we can use these to link to more pages in our website, or other sites we like. For now, they will be placeholders so that we can style them. 

To use the `a` tag, we need to give it a special attribute called `href`. For example:
```
<a href="URL goes here">This text will show up as a link on the page.</a>
```

The `href` attribute is used to specify which URL the `a` tag links to. It's important to set this to something in our placeholders, because links that don't go anywhere will actually look different on the page. We want to see what links that go somewhere look like. 
````